database/migrations/2021_08_14_020237_create_meetings_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMeetingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('meetings', function (Blueprint $table) {
+            $table->id();
+            $table->dateTime('started_at', $precision = 0);
+            $table->dateTime('finished_at', $precision = 0);
+            $table->integer('meetingable_id');
+            $table->string('meetingable_type');
+            $table->foreignId('public_employee_id')->constrained();
+            $table->foreignId('office_id')->constrained();
+            $table->foreignId('reason_id')->constrained();
```

Comment:
No estoy consiguiendo ver dónde colocas al "visitante"

Reply:
Basically, `meetingable_type` would take one of the these values: `visitor` or `public_employee`. That's a Polymorphic Relationship. Since the requirement needs the visitor itself and public employee to be a visitor.